.github/workflows/publish.yml

```
@@ -48,22 +49,30 @@ jobs:
           go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.16.0
       - name: get-version
         run: |
+          COMMIT="$(git rev-parse --short HEAD)"
           if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
               VERSION="${{ github.event.inputs.version }}"
+              IMAGE_VERSION=$VERSION
           elif [[ "$GITHUB_REF_TYPE" == "tag" ]]; then
               VERSION=$GITHUB_REF_NAME
+              IMAGE_VERSION=$VERSION
           else
               VERSION=""
+              IMAGE_VERSION="latest"
           fi
-          COMMIT="$(git rev-parse --short HEAD)"
           PKG_VERSION=${VERSION:-dev-$COMMIT}
           echo "VERSION=$VERSION" >> $GITHUB_ENV
           echo "PKG_VERSION=$PKG_VERSION" >> $GITHUB_ENV
+          echo "IMAGE_VERSION=$IMAGE_VERSION" >> $GITHUB_ENV
       - name: build and package disruptor
         env:
+          VERSION: ${{ env.VERSION }}
           PKG_VERSION: ${{ env.PKG_VERSION }}
         run: |
-          ./build-package.sh -v $PKG_VERSION all
+          if [[ ! -z $VERSION ]]; then 
```

Small readability nit
```suggestion
          if [[ -n $VERSION ]]; then 
```